compaction/task_manager_module.cc

```diff
@@ -7,6 +7,33 @@
  */
 
 #include "compaction/task_manager_module.hh"
+#include "replica/database.hh"
+
+namespace std {
+
+std::ostream& operator<<(std::ostream& os, const api::table_info& ti) {
+    return os << "table{name=" << ti.name << ", id=" << ti.id << "}";
+}
+
+} // namespace std
```

Comment:
I see no reason this would be implemented under `compaction/`.
there's already an implementation in api/storage_service.cc, no?
You can just declare it in the header file.